.env

```diff
@@ -1,3 +1,3 @@
-WEB_SITE_URL = "https://tech-stack.io"
-CAREER_URL = "https://career.tech-stack.com"
+MAIN_PAGE_URL = "https://traineeautomation.azurewebsites.net"
+ADD_USER_PAGE_URL = "https://traineeautomation.azurewebsites.net/Forms/AddUser"
```

Comment:
it might be better to create something like a builder that builds a url based on the MAIN_PAGE_URL

Reply:
Agreed